pkg/reconciler/v1alpha1/route/traffic/traffic.go

```diff
@@ -27,6 +29,10 @@ import (
 // DefaultTarget is the unnamed default target for the traffic.
 const DefaultTarget = ""
 
+type scheme string
```

Comment:
is this type necessary?

Reply:
Possibly not. However, I wanted to make it clear the set of schemes that can be passed through and the format that they should be passed in (i.e. "http" and not "HTTP" or "http://").

Reply:
FYI I can pass any string literal to `subrouteURL` the way Go typing works.

Reply:
Sounds like changing this to just a string default is preferred. Updated.